### ISSUE TITLE
Spring Cache Manager Default TTL and MaxIdleTime

### ISSUE THREAD

**Issue by rshibley** [redisson/redisson#1911]
How can we provide a default TTL and MaxIdleTime to a set of Regions? 
Specifically, I'm using Hibernate with Redisson and would like to have to Set of Cache Types
1- Domain ( TTL 30 Mins )
2- Constants ( TTL 1 Day )

Right now i would have to add every domain class manually in the SpringRedissonConfig. 

```
CacheManager cacheManager(RedissonClient redissonClient) {
        Map<String, CacheConfig> config = new HashMap<>();
        // create "testMap" cache with ttl = 24 minutes and maxIdleTime = 12 minutes
        CacheConfig configItem = new CacheConfig(TimeUnit.MINUTES.toMillis(60), TimeUnit.MINUTES.toMillis(30));
        config.put("domainCache",configItem);
        return new RedissonSpringCacheManager(redissonClient, config);
    }
```

**Comment by jayeshnazre** [comment]
I have a similar problem. But I am still struggling to find how to set the TTL and MaxIdle time for my entities
I am using 
Hibernate 5.3.7, Spring boot 2.1.2, Redisson 3.10.2 version. 

I have annotated my entity beans with 
@Cacheable(cacheNames = "com.jhipster.demo.blog.domain.Tag")
@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)

So I am using a default region and I also tried to set the ttl, max time via code as well as via application.properties file (spring boot auto detects the application.properties key=value sets)

In java code I have the following code snippet
@Configuration
@EnableCaching 
public class CacheConfig {
    @Bean
    CacheManager cacheManager(RedissonClient redissonClient) throws IOException {
        config.put("com.jhipster.demo.blog.domain.Tag", 
        		new CacheConfig(2*60*1000, 3*60*1000));
        return new RedissonSpringCacheManager(redissonClient, config);
    }

}

I also looked at the following link -- 
https://github.com/redisson/redisson/tree/master/redisson-hibernate

And at the bottom I see default values in XML so I used that and thought of putting the same in my application.properties as below (NOTE: since I am using spring boot I have to prefix the same with - "spring.jpa.properties." 

spring.jpa.properties.hibernate.cache.redisson.entry.eviction.max_entries=2000
spring.jpa.properties.hibernate.cache.redisson.entry.eviction.time_to_live=120000
spring.jpa.properties.hibernate.cache.redisson.entry.eviction.max_idle_time=180000

Any help on this will help. I will try to open a new issue on this as well
